opnsense_api/util/__init__.py

```diff
@@ -6,13 +6,6 @@ class Constants(object):
     HTTP_SUCCESS = [200, 201, 202, 203, 204, 205, 206, 207]
 
 
-class AliasType(Enum):
-    HOST = "host"
-    NETWORK = "network"
-    PORT = "port"
-    URL = "url"
```

Comment:
Was there a reason `AliasType` was removed entirely?

Reply:
I eventually discovered `LONG_NAME_TO_TYPE_DICT`, but I'm not really sure that's an improvement.